llvm/tools/repo2obj/R2OELFSymbolTable.h

```diff
@@ -338,7 +344,8 @@ SymbolTable<ELFT>::firstNonLocal(std::vector<Value *> const &OrderedSymbols) {
     auto const Step = Count / 2;
     std::advance(It, Step);
 
-    if ((*It)->linkage() == pstore::repo::linkage_type::internal) {
+    if ((*It)->linkage() == pstore::repo::linkage_type::internal ||
```

Comment:
As above. Better not to call `(*It)->linkage()` twice, esp. inside a loop.

Reply:
Thanks, Fixed it.